pkgs/http_client_conformance_tests/lib/src/response_body_server.dart

```diff
@@ -20,6 +20,10 @@ void hybridMain(StreamChannel<Object?> channel) async {
       await request.drain<void>();
       request.response.headers.set('Access-Control-Allow-Origin', '*');
       request.response.headers.set('Content-Type', 'text/plain');
+      if (request.requestedUri.pathSegments.isNotEmpty &&
+          request.requestedUri.pathSegments.last == 'length') {
+        request.response.contentLength = 'Hello World!'.length;
```

Comment:
[nit] how about adding a variable instead of repeating the content string.

Reply:
Done.